docs/source/common_issues.rst

```diff
@@ -808,3 +808,78 @@ not necessary:
     class NarrowerArgument(A):
         def test(self, t: List[int]) -> Sequence[str]:  # type: ignore[override]
             ...
+
+TypeVars are not the same
+------------------------------------------------
+
+As discussed in the issue
+`TypeVars should be allowed to be the same <https://github.com/python/mypy/issues/7864>`_,
+TypeVars are *not* equivalent even if they are identically defined.
+
+Here is an example using two TypeVars that are unconstrained and unbound:
+
+.. code-block:: python
+
+    from typing import Callable, Iterable, TypeVar
+
+    A = TypeVar("A")
+    B = TypeVar("B")
+
+    def identity(x: A) -> A:
+        return x
+
+    # ok
+    def _map(queue: Iterable[B], function: Callable[[B], A]) -> Iterable[A]:
```

Comment:
Nit: leave out the underscore prefixes, since they serve no function in a code example.